pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -174,7 +200,7 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig) (*bootst
 		etcdEndpoints[i] = fmt.Sprintf("https://etcd-%d.%s:2379", i, installConfig.ClusterDomain())
 	}
 
-	releaseImage := defaultReleaseImage
+	releaseImage := defaultReleaseImage()
```

Comment:
Now that `defaultReleaseImage()` can spit out logs, I think we want:

```go
var releaseImage string
if ri, ok := os.LookupEnv("OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE"); ok && ri != "" {
  logrus.Warn("Found override for ReleaseImage. Please be warned, this is not advised")
  releaseImage = ri
} else {
  releaseImage = defaultReleaseImage()
}
```

If we drop the prefix handling from `defaultReleaseImage` (I'm not sure what it's for), then we could also move the logging out of that function and have:

```go
var releaseImage string
if ri, ok := os.LookupEnv("OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE"); ok && ri != "" {
  logrus.Warn("Found override for ReleaseImage. Please be warned, this is not advised")
  releaseImage = ri
} else {
  releaseImage = defaultReleaseImage()
  logrus.Debugf("Using internal constant for release image %s", releaseImage)
}
```

Reply:
Makes sense.

Prefix is to create an unambiguous replacement - the person doing the replacing needs to scan the binary and replace a long chunk of output - we don't want to catch PREFIX accidentally in other parts of the code, so it's the padding around it plus the prefix that matter.

We could do the constant as `\x00PREFIXXXXXXXXX\x00` I guess